### DISCUSSION TITLE
Giving Myself a Round of Applause

### DISCUSSION THREAD

**Discussion by NishantCoder108** [Cyfrin/foundry-full-course-cu#2892] [General]
I’ve finally understood the basics of blockchain .
Now, I’m excited to start learning how to write my first smart contract in Solidity. 

Thanks @PatrickAlphaC

**Comment by EngrPips** [comment]
Keep crushing it, Friend.

**Comment by chainflash** [comment]
You're doing great, keep up the good work, and don't get tired.

**Comment by LordAdhaar** [comment]
LESSGGOOOO, congrats ser

**Comment by NishantCoder108** [comment]
Finally , I deployed my first contract on a testnet, and I’m very happy to have written and deployed a contract on an actual network like testnet. The exciting part is that by simply switching to the mainnet, the contract can work in the real world. Wow!

Thank you!

**Reply by NishantCoder108** [reply]
💪 Successfully deployed a smart contract on the zkSync testnet network.

**Reply by EngrPips** [reply]
LFG  🚀

**Reply by NishantCoder108** [reply]
I’ve finally mastered the core fundamentals of Solidity syntax and concepts.

Now, I’m starting Foundry Fundamentals.

Thanks again, @PatrickAlphaC!

**Reply by EngrPips** [reply]
Awesome, Keep going.

**Comment by Ifra001** [comment]
GREAT WORK!!

**Reply by EngrPips** [reply]
Indeed.
